flink-ai-flow/ai_flow/client/__init__.py

```diff
@@ -307,20 +307,19 @@ def get_artifact_by_name(artifact_name) -> Optional[ArtifactMeta]:
     return get_ai_flow_client().get_artifact_by_name(artifact_name)
 
 
-def register_artifact(name: Text, data_format: Text = None, description: Text = None,
-                      batch_uri: Text = None, stream_uri: Text = None,
+def register_artifact(name: Text, artifact_type: Text = None, description: Text = None,
+                      uri: Text = None,
                       create_time: int = None, update_time: int = None,
                       properties: Properties = None) -> ArtifactMeta:
-    return get_ai_flow_client().register_artifact(name, data_format, description, batch_uri, stream_uri, create_time,
+    return get_ai_flow_client().register_artifact(name, artifact_type, description, uri, create_time,
                                                   update_time, properties)
 
 
-def update_artifact(artifact_name: Text, data_format: Text = None, description: Text = None,
-                    batch_uri: Text = None, stream_uri: Text = None,
+def update_artifact(artifact_name: Text, artifact_type: Text = None, description: Text = None,
+                    uri: Text = None,
                     update_time: int = None, properties: Properties = None) -> Optional[ArtifactMeta]:
```

Comment:
`update_time` could be removed because users doesn't need to assign this fields value when updating artifact.